frost-core/src/scalar_mul.rs

```diff
@@ -8,9 +8,30 @@ use std::{
 
 use crate::{Ciphersuite, Element, Field, Group, Scalar};
 
+/// Calculates the quotient of `self` and `rhs`, rounding the result towards positive infinity.
+///
+/// # Panics
+///
+/// This function will panic if `rhs` is 0 or the division results in overflow.
+///
+/// This function is similar to `div_ceil` that is [available on
+/// Nightly](https://github.com/rust-lang/rust/issues/88581).
+///
+// TODO: remove this function and use `div_ceil()` instead when `int_roundings`
+// is stabilized.
+const fn div_ceil(lhs: usize, rhs: usize) -> usize {
+    let d = lhs / rhs;
+    let r = lhs % rhs;
+    if r > 0 && rhs > 0 {
+        d + 1
+    } else {
+        d
+    }
```

Comment:
Fun fact: this is equivalent to `(lhs + (rhs - 1)) / rhs`. No need to change it though.

Reply:
This is literally ported from ruststd and i hope it will go away soon 😆